src/utils/email.js

```diff
@@ -0,0 +1,25 @@
+import nodemailer from "nodemailer";
+import nodemailerSendgrid from "nodemailer-sendgrid";
+import dotenv from "dotenv";
+
+dotenv.config();
+const transport = nodemailer.createTransport(
+  nodemailerSendgrid({
+    apiKey: process.env.SENDGRID_API_KEY
+  })
+);
+
+const sendConfirmationEmail = async (user) => {
+  const url = `${process.env.APP_URL}/me`;
+  await transport.sendMail({
+    from: "strikers668@gmail.com",
```

Comment:
It is good to add the sender email as an env variable as it can change. This would help not going back into editing the codebase.